OloEngine/src/OloEngine/Scripting/C#/ScriptEngine.cpp

```
@@ -616,3 +619,108 @@ namespace OloEngine
         return true;
     }
 } // namespace OloEngine
+
+#else // !OLO_ENABLE_CSHARP_SCRIPTING — stub implementation
+
+namespace OloEngine
+{
+    // Minimal scene-context management so LuaScriptGlue can call GetSceneContext()
+    static Scene* s_SceneContext = nullptr;
```

Clang (14.0.6)

[warning] variable 's_SceneContext' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)